resources/conf/config.toml

```diff
@@ -19,6 +19,17 @@
   securedListenerPort = 9095
   systemHost = "localhost"
 
+[router.ratelimit]
+  enabled = true
+  host = "rate-limiter"
+  port = 8091
+  requestTimeoutInMillis= 80
+  failureModeDeny = false
+  keyFilePath = "/home/wso2/security/keystore/mg.key"
+  certFilePath = "/home/wso2/security/keystore/mg.pem"
+  caCertFilePath = "/home/wso2/security/truststore/rate-limiter.pem"
+  sSLCertSANHostname = ""
```

Comment:
I think we can remove this part. Adding enabled true is enough.

```suggestion
```

Reply:
In this case, rate limits are enabled in integration tests, right? But it got passed.
The reason to add this config is that we wanted to enable this by default with config.toml. Because if someone tryout rate limiting it will not work, even though we have added Redis and rate limiter services in docker compose setup. @VirajSalaka wdht?

Maybe we can enable this in k8s config too.

Reply:
+1